### PR TITLE
pangeo notebook env with regionmask

### PR DESCRIPTION
**Pull request by aaronspring**
Content:
-  [x]  using `pangeo_notebook`
-  [x]  `regionmask` in environment
-  [ ]  more timesteps and variables nicely loaded: currently a bit bumby flow
-  [ ]  daily time for temp and precip not equal
-  [ ]  check all notebooks dependencies and add to env
-  [x]  enable `regionmask` with `rasterize`: `ds.lon-180` for speedup
-  [ ]  align `csv` filenames to previous standard
-  [x]  integrate all country shapefiles available into yaml

checkout: https://binder.pangeo.io/v2/gh/aaronspring/covid19-examples-and-docs/AS_pangeo_notebook

### CODE REVIEW

**Comment by aaronspring** [comment]
`regionmask` doesnt scale with dask.

File: examples/country_config.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+UK:
```
Comment (aaronspring) [inline examples/country_config.yaml:1]: @kaedonkers I set up this yaml file that will match all the different args required to process the different shapefiles with `regionmask`

Comment (kaedonkers) [reply examples/country_config.yaml:1]: Nice! I am doing something similar with [`shapefile_attributes.json`](https://github.com/informatics-lab/covid-19_shapecutter/blob/semi-auto/shapefile_attributes.json) in the latest version of my code. I think I will take some lessons from your implementation...

File: examples/country_config.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  keywords: [United Kingdom, England, uk, england]  # not politically correct
+  base: https://metdatasa.blob.core.windows.net/covid19-response/shapefiles/England
```
Comment (kaedonkers) [inline examples/country_config.yaml:3]: There are separate shapefiles for the UK and England. The UK ones are in `https://metdatasa.blob.core.windows.net/covid19-response/shapefiles/UK` and are a combination of England, Scotland, Wales and N. Ireland reporting regions as geometries. I would strongly encourage making this clear, possibly even removing the England shapefile link to stop any confusion.

File: examples/country_config.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+UK:
+  keywords: [United Kingdom, England, uk, england]  # not politically correct
```
Comment (kaedonkers) [inline examples/country_config.yaml:2]: 😝

File: examples/country_config.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  base: https://metdatasa.blob.core.windows.net/covid19-response/shapefiles/England
+  name: Counties_and_Unitary_Authorities_April_2019_Boundaries_EW_BUC
```
Comment (kaedonkers) [inline examples/country_config.yaml:4]: In line with the comment above - `UK_covid_reporting_regions`

File: examples/country_config.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  from_geopandas_args:
+    names: ctyua19nm
```
Comment (kaedonkers) [inline examples/country_config.yaml:6]: `name`

File: examples/country_config.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    abbrevs: ctyua19cd
```
Comment (kaedonkers) [inline examples/country_config.yaml:7]: `code`

**Pull request closed without merging**